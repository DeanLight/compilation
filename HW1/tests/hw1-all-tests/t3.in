%integers
0
000
123
1234 5678
00003
+0	-000
+123
21+12%tokens don't need whitespace
0987654321
-95-87%and again
