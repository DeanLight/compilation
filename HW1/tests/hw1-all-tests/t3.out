1 COMMENT %integers
2 INTEGER 0
3 INTEGER 000
4 INTEGER 123
5 INTEGER 1234
5 INTEGER 5678
6 INTEGER 00003
7 INTEGER +0
7 INTEGER -000
8 INTEGER +123
9 INTEGER 21
9 INTEGER +12
9 COMMENT %tokens don't need whitespace
10 INTEGER 0987654321
11 INTEGER -95
11 INTEGER -87
11 COMMENT %and again
12 EOF 
